### ISSUE TITLE
keras_parikh_entailment hyperparameters to reproduce the results 

### ISSUE THREAD

**Issue by aagohary** [explosion/spaCy#825]
would you please share the values of the hyperparameters that produce the 85% accuracy on the dev set? My best dev accuracy so far is 80%

**Comment by honnibal** [comment]
I've been having trouble with this too. I think I might have introduced a bug.

**Comment by aagohary** [comment]
OK Just a followup question (might help with reproducing the results):
I don't see how you make sure you don't add the attention scores of the padded zeros to the denominator of the softmax normalization (eq. 2 in the paper).

**Comment by ines** [comment]
Merging this with #1445!
